python/ycm/client/command_request.py

```diff
@@ -69,9 +71,50 @@ def RunPostCommandActionsIfNeeded( self ):
         vimsupport.JumpToLocation( self._response[ 'filepath' ],
                                     self._response[ 'line_num' ],
                                     self._response[ 'column_num' ] )
-    elif 'message' in self._response:
-      vimsupport.EchoText( self._response['message'] )
+    elif self._is_fixit_command:
```

Comment:
This function now seems to be in need of refactoring; instead of inlining all the code for the separate cases of command, we should pull those out into separate functions.